stdlib/public/core/SIMDVector.swift

```diff
@@ -1157,60 +1197,112 @@ extension SIMD where Scalar: FixedWidthInteger {
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpAdd(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : Differentiable & BinaryFloatingPoint,
+          Scalar.TangentVector : BinaryFloatingPoint,
+          Self.TangentVector.Scalar == Scalar.TangentVector)
   public static func -(lhs: Scalar, rhs: Self) -> Self {
     return Self(repeating: lhs) - rhs
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpMultiply(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : BinaryFloatingPoint & Differentiable,
+          Self.TangentVector == Self,
+          Scalar.TangentVector == Scalar)
   public static func *(lhs: Scalar, rhs: Self) -> Self {
     return Self(repeating: lhs) * rhs
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpDivide(lhs:rhs:)
+  where Self : Differentiable,
+  Self.TangentVector : SIMD,
+  Scalar : BinaryFloatingPoint & Differentiable,
+  Self.TangentVector == Self,
+  Scalar.TangentVector == Scalar)
   public static func /(lhs: Scalar, rhs: Self) -> Self {
     return Self(repeating: lhs) / rhs
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpAdd(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : Differentiable & BinaryFloatingPoint,
+          Scalar.TangentVector : BinaryFloatingPoint,
+          Self.TangentVector.Scalar == Scalar.TangentVector)
   public static func +(lhs: Self, rhs: Scalar) -> Self {
     return lhs + Self(repeating: rhs)
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpAdd(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : Differentiable & BinaryFloatingPoint,
+          Scalar.TangentVector : BinaryFloatingPoint,
+          Self.TangentVector.Scalar == Scalar.TangentVector)
   public static func -(lhs: Self, rhs: Scalar) -> Self {
     return lhs - Self(repeating: rhs)
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpMultiply(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : BinaryFloatingPoint & Differentiable,
+          Self.TangentVector == Self,
+          Scalar.TangentVector == Scalar)
   public static func *(lhs: Self, rhs: Scalar) -> Self {
     return lhs * Self(repeating: rhs)
   }
   
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(vjp: _vjpDivide(lhs:rhs:)
+    where Self : Differentiable,
+          Self.TangentVector : SIMD,
+          Scalar : BinaryFloatingPoint & Differentiable,
+          Self.TangentVector == Self,
+          Scalar.TangentVector == Scalar)
   public static func /(lhs: Self, rhs: Scalar) -> Self {
     return lhs / Self(repeating: rhs)
   }
   
-  @_transparent
-  public static func +=(lhs: inout Self, rhs: Self) {
```

Comment:
In order to make SIMD conform to AdditiveArithmetic, I had to remove both of these implementations as two protocols cannot both have default implementations. Let me know if this is okay, or if there is something else we should do.